codecoverage.py

```diff
@@ -135,7 +147,28 @@ def _is_in_platforms_task(t):
             if not any(suite in t['task']['metadata']['name'] for t in test_tasks):
                 warnings.warn('Suite %s not found' % suite)
 
-    for i, test_task in enumerate(test_tasks):
+    download_tasks = {}
+
+    for test_task in test_tasks:
+        status = test_task['status']['state']
+        assert status in ALL_STATUSES, "State '{}' not recognized".format(status)
+
+        while status not in FINISHED_STATUSES:
+            time.sleep(60)
```

Comment:
We should print something here, otherwise it will be confusing for users.
E.g.:
```
         sys.stdout.write('\rWaiting for task {} to finish...'.format(test_task['status']['taskId']))
         sys.stdout.flush()
```